OrcaCollective/techbloc-airflow
# Initial Airflow Setup

This PR adds initial Airflow setup for techbloc. This is heavily borrowed from https://github.com/WordPress/openverse-catalog.

Important files to look at:
- `env.template`
- `justfile`
- `docker-compose*.yml`
- `Dockerfile`

## Testing instructions
1. `just build && just up`, then go to http://localhost:9090 and ensure that you can log in with airflow/airflow
